ruby_event_store/lib/ruby_event_store/client.rb

```diff
@@ -194,6 +194,13 @@ def read
       Specification.new(SpecificationReader.new(repository, mapper))
     end
 
+    # Gets list of stream names where event is stored or linked
+    #
+    # @return [Array<String>] names of streams where event is stored or linked
+    def streams_of(event_id)
+      repository.streams_of(event_id)
```

Comment:
would it make sense to return a collection of Stream objects? Given that we already have them as a kind of value objects.

Reply:
yeah - makes sense